src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -17,17 +17,19 @@
 <kd-help-section>
   <md-input-container class="md-block" md-is-error="ctrl.isNameError()">
     <label>App name</label>
-    <input ng-model="ctrl.name" name="name" namespace="ctrl.namespace" required kd-unique-name
-        ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 500, 'blur': 0 } }">
-    <md-progress-linear class="kd-deploy-form-progress" md-mode="indeterminate"
-        ng-show="ctrl.form.name.$pending">
-    </md-progress-linear>
-    <ng-messages for="ctrl.form.name.$error" role="alert" multiple>
-      <ng-message when="required">Application name is required.</ng-message>
-      <ng-message when="uniqueName">
-        Application with this name already exists within namespace <i>{{ctrl.namespace}}</i>.
-      </ng-message>
-    </ng-messages>
+    <input ng-model="ctrl.name"
+           name="name"
```

Comment:
For the unique name check the debounce is required. I assume this delay affects all validations on the field. 

The check on focus out with debounce is fine and done everywhere like this. However, the continues check after the first failure is new. I have not seen this on any other web page. The check as you type together with debounce might be a bit strange. 

Anyway, we should stick to the default behavior (debounce) of the framework  and leaf it for the UX expert to think about improvements